docs/reference/env-variables.rst

```diff
@@ -0,0 +1,971 @@
+.. meta::
+    :description: Environment variables reference
+    :keywords: AMD, ROCm, environment variables, environment, reference
+
+.. role:: cpp(code)
+   :language: cpp
+
+.. _env-variables-reference:
+
+*************************************************************
+Environment variables reference
+*************************************************************
+
+ROCm common and important Environment Variables
+===============================================
+
+The following table contains the environment variables that are most commonly used in the ROCm software stack. These variables can be useful for simple tasks, like building a ROCm library or running applications on AMD cards.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 70,30
+
+    * - **Environment variable**
+      - **Usage**
+
```

Comment:
Add another column titled "Description" and move the description to this second column. Don't write it along with the environment variable.

Reply:
We had that version and the text not fitting well. The description is better positioned, if it goes under the function or env variable.

examples:
https://github.com/ROCm/HIP/blob/develop/docs/reference/kernel_language.rst

Reply:
@lpaoletti Is it okay to write description in the same column as env variable?